lib/modern_searchlogic/column_conditions.rb

```diff
@@ -102,6 +102,26 @@ def searchlogic_association_suffix_match(method_name)
         end
       end
 
+      def searchlogic_column_boolean_match(method_name)
+        if match = method_name.match(/^not_(.*)/)
+          column_name = match[1]
+          if boolean_column?(column_name)
+            return {:arity => 0, :block => lambda do
+              where(column_name => false)
+            end}
+          end
+        elsif boolean_column?(method_name)
+          return {:arity => 0, :block => lambda do
+            where(method_name => true)
+          end}
+        end
+      end
+
+      def boolean_column?(name)
+        column = columns_hash[name.to_s] || columns_hash[name.to_s.gsub(/^not_/, "")]
```

Comment:
do you need the second clause here? seems redundant since you're pulling the column out of a regexp group

Reply:
Good call. That can definitely go. It's been removed as part of an amend to the original commit in a21b217453bd8cad96144bbe06bd213f7f355ecd